package.json

```diff
@@ -0,0 +1,630 @@
+    "spdx-license-ids": "^3.0.13",
+    "specificity": "^0.4.1",
+    "sprintf-js": "^1.0.3",
+    "ssri": "^9.0.1",
+    "statuses": "^2.0.1",
+    "string_decoder": "^1.3.0",
+    "string-width": "^4.2.3",
+    "string.prototype.trim": "^1.2.7",
+    "string.prototype.trimend": "^1.0.6",
+    "string.prototype.trimstart": "^1.0.6",
+    "strip-ansi": "^6.0.1",
+    "strip-bom": "^3.0.0",
+    "strip-final-newline": "^2.0.0",
+    "strip-indent": "^3.0.0",
+    "strip-json-comments": "^2.0.1",
+    "strnum": "^1.0.5",
+    "strtok3": "^6.3.0",
+    "style-search": "^0.1.0",
+    "stylelint-config-recommended": "^4.0.0",
+    "sugarss": "^2.0.0",
+    "supports-color": "^7.2.0",
+    "supports-preserve-symlinks-flag": "^1.0.0",
+    "svg-tags": "^1.0.0",
+    "symbol-tree": "^3.2.4",
+    "table": "^6.8.1",
+    "tar": "^6.1.13",
+    "tar-fs": "^2.1.1",
+    "tar-stream": "^2.2.0",
+    "text-table": "^0.2.0",
+    "through": "^2.3.8",
+    "to-fast-properties": "^2.0.0",
+    "to-readable-stream": "^1.0.0",
+    "to-regex-range": "^5.0.1",
+    "toidentifier": "^1.0.1",
+    "token-types": "^4.2.1",
+    "tough-cookie": "^4.1.2",
+    "tr46": "^3.0.0",
+    "trim-newlines": "^3.0.1",
+    "trough": "^1.0.5",
+    "tsconfig-paths": "^3.14.2",
+    "tslib": "^2.5.0",
+    "tsutils": "^3.21.0",
+    "type-check": "^0.4.0",
+    "type-fest": "^0.20.2",
+    "typed-array-length": "^1.0.4",
+    "typedarray-to-buffer": "^3.1.5",
+    "typescript": "^4.9.5",
+    "unbox-primitive": "^1.0.2",
+    "unbzip2-stream": "^1.4.3",
+    "unified": "^9.2.2",
+    "unique-filename": "^2.0.1",
+    "unique-slug": "^3.0.0",
+    "unique-string": "^2.0.0",
+    "unist-util-find-all-after": "^3.0.2",
+    "unist-util-is": "^4.1.0",
+    "unist-util-stringify-position": "^2.0.3",
+    "universalify": "^2.0.0",
+    "unpipe": "^1.0.0",
+    "update-browserslist-db": "^1.0.10",
+    "update-notifier": "^6.0.2",
+    "uri-js": "^4.4.1",
+    "url-parse": "^1.5.10",
+    "url-parse-lax": "^3.0.0",
+    "util-deprecate": "^1.0.2",
+    "v8-compile-cache": "^2.3.0",
+    "validate-npm-package-license": "^3.0.4",
+    "validate-npm-package-name": "^4.0.0",
+    "vfile": "^4.2.1",
+    "vfile-message": "^2.0.4",
+    "vm2": "^3.9.14",
+    "w3c-xmlserializer": "^4.0.0",
+    "wcwidth": "^1.0.1",
+    "webidl-conversions": "^7.0.0",
+    "whatwg-encoding": "^2.0.0",
+    "whatwg-mimetype": "^3.0.0",
+    "whatwg-url": "^11.0.0",
+    "which": "^2.0.2",
+    "which-boxed-primitive": "^1.0.2",
+    "which-typed-array": "^1.1.9",
+    "wide-align": "^1.1.5",
+    "widest-line": "^3.1.0",
+    "word-wrap": "^1.2.3",
+    "wrap-ansi": "^7.0.0",
+    "wrappy": "^1.0.2",
+    "write-file-atomic": "^3.0.3",
+    "ws": "^8.13.0",
+    "xdg-basedir": "^4.0.0",
+    "xml-name-validator": "^4.0.0",
+    "xmlchars": "^2.2.0",
+    "xregexp": "^2.0.0",
+    "yallist": "^4.0.0",
+    "yaml": "^1.10.2",
+    "yargs-parser": "^20.2.9",
+    "yauzl": "^2.10.0",
+    "zwitch": "^1.0.5"
+  },
```

Comment:
- [ ] Your project has a lot of unused dependencies and some dependencies that are preventing users to install them with the `npm i` command. This makes your project bloated and heavy, and you should not add dependencies that are not used in the project because of these reasons. Kindly change the dependencies of your `package.json` only with dependencies that are being used in this project and that are not deprecated or unable to install. Please see this [documentation](https://webpack.js.org/guides/getting-started/) for guidance. When I try to run the `npm i` command I get this error message:
 - 
<img width="1034" alt="image" src="https://user-images.githubusercontent.com/61919617/225345621-c7dd0c5c-5b28-4550-ba12-07213bb450bd.png">

Reply:
Thank you for reviewing my code👍🏽. Have a wonderful day 😎